### PR TITLE
[MIG] 12.0 mig fleet vehicle license plate alt

### PR DESCRIPTION
**Pull request by mymage**
Backportig from 13.0

### CODE REVIEW

File: fleet_vehicle_license_plate_alt/models/fleet_vehicle.py [hunk 1]
```diff
@@ -32,4 +32,5 @@ def _name_search(
         rec = self._search(
             expression.AND([domain, args]), limit=limit, access_rights_uid=name_get_uid
         )
-        return models.lazy_name_get(self.browse(rec).with_user(name_get_uid))
+        # return models.lazy_name_get(self.browse(rec).with_user(name_get_uid))
```
Comment (ivantodorovich) [inline fleet_vehicle_license_plate_alt/models/fleet_vehicle.py]:
```suggestion
```

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
FYI you don't need to re-generate readme.rst and html, that's already handled by the oca bot upon merge :)

**Review by ivantodorovich — APPROVED** [review]

File: fleet_vehicle_license_plate_alt/models/fleet_vehicle.py (hunk 1 above)
Comment (mymage) [reply fleet_vehicle_license_plate_alt/models/fleet_vehicle.py]: Hey, you moved before I say ;-)
As I say, I don't know how to use test, so when Travis set an error on it I tried the first thing that traveled on my mind, and it went!
I hope to have the same luck with next modules.

**Comment by mymage** [comment]
> 
> 
> FYI you don't need to re-generate readme.rst and html, that's already handled by the oca bot upon merge :)

Thanks for your tip, but as for comments I am an orderly person so I prefer to hale all aligned :-)

**Review by mamcode — APPROVED** [review]
LGTM, Thanks for backport to v12.0.

**Pull request merged**